scripts/environment.json

```diff
@@ -1,5 +1,8 @@
 {
-    "cipd_package_files": ["scripts/pigweed.json"],
+    "cipd_package_files": [
+        "third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/cipd_setup/pigweed.json",
```

Comment:
The default `pigweed.json` contains some packages which we don't need like `go`? Removing those saved up to 20s per CI build as far as I remember. Do you think it's not worth maintaining it anyway?

Reply:
Pigweed split out cpython for us, they might be willing to split it up even further.

I don't mind removing go, although I think it's possible we'll end up wanting to use some of the go based tools eventually.

The dumb thing is that GitHub re-bootstraps every time; we don't have that problem in our infrastructure so the cost of carrying these is amortized down to nothing.